src/odemis/acq/test/millmng_test.py

```diff
@@ -0,0 +1,332 @@
+
+        # set the current for all the milling settings
+        # minumum current is selected for both milling current and drift correction current
+        # depending on the Virtual Machine and microscope file used
+        cls.probe_current = min(cls.ion_beam.probeCurrent.choices)
+
+        cls.ccd.exposureTime.value = 0.1  # s, go fast (but not too fast, to still get some signal)
+        # opm = acq.path.OpticalPathManager(model.getMicroscope())  # TODO ensures that the align lens is active and stage tilt is 0°
+        fs1 = stream.FluoStream("fluo1", cls.ccd, cls.ccd.data,
+                                cls.light, cls.light_filter, focuser=cls.focus)
+        fs1.excitation.value = sorted(fs1.excitation.choices)[0]
+
+        cls.acq_streams = [fs1]
+
+        current_stage_pos = cls.stage.position.value
+        logging.debug("current stage position %s", current_stage_pos)
+
+        target_position_1 = {"x": current_stage_pos["x"],
+                             "y": current_stage_pos["y"],
+                             "z": current_stage_pos["z"]}
+
+        target_position_2 = {"x": current_stage_pos["x"] - 15e-06,
+                             # -delta moves the horizontal stage in left direction
+                             "y": current_stage_pos["y"] + 15e-06,
+                             # +delta moves the vertical stage in upward direction
+                             "z": current_stage_pos["z"]}
+
+        cls.target_position = [target_position_1, target_position_2]
+
+        try:
+            cls.beam_angle = cls.stage.getMetadata()[model.MD_ION_BEAM_TO_SAMPLE_ANGLE]
+        except KeyError:
+            raise ValueError("The stage is missing an ION_BEAM_TO_SAMPLE_ANGLE metadata.")
+
+        cls.sites = []
+        for i in range(0, len(cls.target_position)):
+            feature = CryoFeature('object_' + str(i), cls.target_position[i]['x'], cls.target_position[i]['y'],
+                                  cls.target_position[i]['z'], milling_angle=0)
+            cls.sites.append(feature)
+
+        cls.feature_post_status = FEATURE_ROUGH_MILLED
+
+        # TODO support different drift correction current than the milling current in milling manager
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.stage.moveAbs(cls.target_position[0]).result()
+        if TEST_NOHW:
+            cls.patch_obj.stop()
+            # testing.stop_backend()
+
+    def setUp(self):
+        # reset to current stage position
+        self.stage.moveAbs(self.target_position[0]).result()
+
+    def test_milling_settings(self):
+        """
+        Test the values in milling settings with predefined values
+        """
+        milling_setting_1 = MillingSettings(name='rough_milling_1', current=self.probe_current, horizontal_fov=35e-6,
+                                            roi=(0.5, 0.5, 0.8, 0.8),
+                                            pixel_size=(3.41796875e-08, 3.41796875e-08), beam_angle=self.beam_angle,
```

Comment:
Why such value for the pixel size? You decide it, and it's quite arbitrary (it actually depends on the probe size, which depends on the probe current). One thousands for the FoV would probably be fine for here => 35e-9, or even 10e-9. Something more round brings the feeling that it's arbitrary.